docs/references/nextjs/route-handlers.mdx

```diff
@@ -110,7 +110,7 @@ export async function POST(req:NextRequest) {
 
   const params = { firstName: 'John', lastName: 'Wick' };
 
-  const user = await clerkClient.users.updateUser(userId, params);
+  const user = await clerkClient().users.updateUser(userId, params);
```

Comment:
`clerkClient` as a singleton is not deprecated in favor of `clerkClient()` which reads dynamic keys according to it's runtime https://github.com/clerk/javascript/pull/3617

Reply:
is this true just for next.js ? or all sdk's

Reply:
Only for `clerkClient` exported from `@clerk/nextjs`.

Reply:
ahh, this will require many more changes in the docs - I can work on a PR once https://github.com/clerk/javascript/pull/3617 is merged